### PR TITLE
Start integrating voice token

### PR DESCRIPTION
**Pull request by josejulio**

### CODE REVIEW

File: src/ballots/vote.cpp [hunk 1]
```diff
@@ -72,11 +72,17 @@ namespace hypha
...
-        hypha::voice::accounts v_t(hvoiceContract, voter.value);
+        hypha::accounts acnts(hvoiceContract, voter.value);
+        auto account_index = acnts.get_index<name("bykey")>();
 
         asset voiceToken = daoSettings->getOrFail<asset>(common::VOICE_TOKEN);
 
-        auto v_itr = v_t.find(voiceToken.symbol.code().raw());
+        auto v_itr = account_index.find(
+            account::build_key(
+                name(""), // TODO: use dao name
```
Comment (josejulio) [inline src/ballots/vote.cpp:82]: @Gerard097 what would be the easier way to get the name?

Comment (josejulio) [reply src/ballots/vote.cpp:82]: thanks!

File: src/proposals/proposal.cpp [hunk 2]
```diff
@@ -183,8 +183,15 @@ namespace hypha
...
         float alignmentFactor = m_daoSettings->getOrFail<int64_t>(VOTING_ALIGNMENT_FACTOR_X100) / 100.0f;
 
-        hypha::voice::stats stats_t(voiceContract, voiceToken.symbol.code().raw());
-        auto stat_itr = stats_t.find(voiceToken.symbol.code().raw());
+        hypha::stats statstable(voiceContract, voiceToken.symbol.code().raw());
+        auto stats_index = statstable.get_index<name("bykey")>();
+
+        auto stat_itr = stats_index.find(
+            currency_stats::build_key(
+                name(""), // use tenant here (as a name)
```
Comment (n13) [inline src/proposals/proposal.cpp:191]: Wouldn't you be able to get the tenant from the proposal / document that was being voted on?

Comment (josejulio) [reply src/proposals/proposal.cpp]: That's where I got it from in the end. The proposal object has a reference to the dao setting.

**Pull request merged**